src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php

```diff
@@ -460,6 +519,22 @@ protected function extractFilesFromDataArray(&$data)
         return $files;
     }
 
+    /**
+     * If enabled, encrypt cookie values for request.
+     *
+     * @return array
+     */
+    protected function prepareCookiesForRequest()
+    {
+        if (! $this->encryptCookies) {
+            return $this->defaultCookies;
```

Comment:
what would happen here if I needed my own cookies?

Reply:
This method is only effects the code if you use these new methods and call one of the HTTP method helpers. As noted in the PR, if you have your own encryption/cookies, you can continue testing as you have before using `call` directly.

Reply:
It makes sense, thanks for your reply. 

I am still a bit lost with the implementation tho. For instance, if I can `withCookies`, should I expect to send the `default` ones too?. 

Is it expected to removed the default? 

This is my main doubt by looking at the changes.